src/Api/Controllers/GroupsController.cs

```diff
@@ -88,7 +104,14 @@ public async Task<IEnumerable<Guid>> GetUsers(string orgId, string id)
     {
         var idGuid = new Guid(id);
         var group = await _groupRepository.GetByIdAsync(idGuid);
-        if (group == null || !await _currentContext.ManageGroups(group.OrganizationId))
+        if (group == null)
+        {
+            throw new NotFoundException();
+        }
+
+        var authorizationResult =
```

Comment:
and likely for all reads

Reply:
Resolving this to remove duplicate comments. I'll address this per the outcome of the above comment thread.